From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aikar <aikar@aikar.co>
Date: Tue, 29 Apr 2014 20:27:43 -0400
Subject: [PATCH] All blocks extinguish at same rate

Previously "Flammable" blocks would burn for ages, this makes them burn out the same as non flammable.
This is so that blocks that have ignite/burn events cancelled will actually burn out.

diff --git a/src/main/java/net/minecraft/world/level/block/FireBlock.java b/src/main/java/net/minecraft/world/level/block/FireBlock.java
index 2c7847aebabe14da44b9a42f5ecae77858fb9dd3..a9711197daa1b7989b7eaf2b991e91de8a063e2d 100644
--- a/src/main/java/net/minecraft/world/level/block/FireBlock.java
+++ b/src/main/java/net/minecraft/world/level/block/FireBlock.java
@@ -196,7 +196,7 @@ public class FireBlock extends BaseFireBlock {
                         return;
                     }
 
-                    if (i == 15 && random.nextInt(4) == 0 && !this.canBurn(world.getBlockState(pos.below()))) {
+                    if (i == 15 && random.nextInt(4) == 0 /*&& !this.canBurn(world.getBlockState(pos.below()))*/) { // EMC
                         this.fireExtinguished(world, pos); // CraftBukkit
                         return;
                     }
